src/components/ImageBanner/ImageBanner_spec.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import 'jest-styled-components'
+import { shallow, mount } from 'enzyme'
+
+import ImageBanner, { BackgroundImage } from './ImageBanner'
+import BannerTitle from '../BannerTitle/BannerTitle'
+import BannerSubtitle from '../BannerSubtitle/BannerSubtitle'
+
+describe('ImageBanner', () => {
+  it('should render', () => {
+    const component = renderer.create(<ImageBanner />).toJSON()
+    expect(component).toMatchSnapshot()
+  })
+
+  it('should render a <BannerTitle />', () => {
+    const wrapper = shallow(<ImageBanner />)
+    expect(wrapper.find(BannerTitle).length).toBe(1)
```

Comment:
Jest's expect library has a  `toHaveLength()` method that's good for checking length